### PR TITLE
Initial site files created, rudimentary UI laid out

### PR DESCRIPTION
**Pull request by dvavs**
1. Added initial site directories and files
2. Fleshed out the html doc to give structure similar to what we put in the project proposal
3. Linked jQuery, project JS, project CSS, and Bootstrap (for now until we nail down which CSS framework to use)
4. Applied some basic styles to style.css to help visualize the project better
5. Created a test alert to make sure the JS was linked correctly

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/* Just some random styles to help show the bare html layout generated so far */
+body {
+    text-align: center;
+}
+
+.container {
+    min-height: 100vh;
+    margin: -60px;
```
Comment (ghost) [inline assets/css/style.css]: These need to be changed to margin-bottom, padding-bottom.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,126 @@
...
+        </div>
+    </div>
+
+    <!-- Sticky footer that will never come above the bottom of the page -->
+    <footer>
+        <p>Team Firefoxes</p>
+    </footer>
+
+
+    <!-- link to jQuery -->
```
Comment (ghost) [inline index.html]: Recommend putting all libraries at the top.

Comment (dvavs) [reply index.html]: Done!

File: assets/css/style.css (hunk 1 above)
Comment (dvavs) [reply assets/css/style.css]: Done!

**Review by ghost — APPROVED** [review]

**Pull request merged**